test/grpc.js

```diff
@@ -46,7 +46,7 @@ describe('grpc', function() {
     });
   });
 
-  describe.skip('metadataBuilder', function() {
+  describe('metadataBuilder', function() {
```

Comment:
Yes, it runs and that's fine. MetadataBuilder exists but it ignores A/B testing parameters at all right now, and this test case verifies it successfully behaves.